server/src/main/resources/static/old-console/js/jquery.dataTables-1.8.2.js

```diff
@@ -1,24 +1,19 @@
 /*
- * File:        jquery.dataTables.js
```

Comment:
We can't change a license header like this. The policy on https://www.apache.org/legal/resolved.html says that 3-clause BSD is OK to include, though. So I guess we should keep this header the way it is and add an exception to rat. If possible, the exception should be scoped to just this file.